task-2/client.py

```diff
@@ -0,0 +1,76 @@
+import requests
+import json
+
+from dataclasses import dataclass, field, asdict
+from dataclasses_json import DataClassJsonMixin, config
+
+from typing import List, Dict
+
+import marshmallow as mm
+
+
+@dataclass
+class Interface(DataClassJsonMixin):
+    name: str = ""
+    description: str = ""
+    type: str = ""
+    enabled: bool = False
+    link_up_down_trap_enable: str = field(default="", metadata=config(field_name="link-up-down-trap-enable"))
+    ietf_ip_ipv4: Dict[str, List[Dict[str, str]]] = field(default_factory=Dict, 
+                    metadata=config(
+                            field_name="ietf-ip:ipv4",
+                            mm_field=mm.fields.Dict(
+                                key=mm.fields.Str(),
+                                values=mm.fields.List(
+                                    mm.fields.Dict(
+                                        keys=mm.fields.Str(),
+                                        values=mm.fields.Str()
+                                    )
+                                ),
+                            )
+                        )
+                    )
```

Comment:
@tomowex use another dataclass for this as well.. you have two different values here it will look more clean

Also we talked about to wrap this with another dataclass that would be list of Interface dataclass so you can let marshmallow take care of all.